front/app/containers/Authentication/SignUpIn/SignUpInModal/SignIn/PasswordSignin.tsx

```diff
@@ -97,8 +97,8 @@ const PasswordSignin = ({
   const [signInError, setSignInError] = useState<string | null>(null);
 
   const [hasEmptyPasswordError, setHasEmptyPasswordError] = useState(false);
-  let emailInputElement = useRef<HTMLInputElement | null>(null);
-  let passwordInputElement = useRef<HTMLInputElement | null>(null);
+  const emailInputElement = useRef<HTMLInputElement | null>(null);
+  const passwordInputElement = useRef<HTMLInputElement | null>(null);
```

Comment:
It was also changed automatically by the linter or something.

Reply:
These are good